recipes/ipycytoscape/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ipycytoscape" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 74e3c72ac2d48aba61cffb5db62f0ea15dfd3cab7075d469d444c7a01891162a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
+    - pip
+    - jupyter
```

Comment:
I don't think there should be a build section for this recipe

Reply:
Right. Thanks for the feedback @scopatz :)